openqabot/types/incident.py

```diff
@@ -19,6 +19,7 @@ def __init__(self, incident):
         self.id = incident["number"]
         self.rrid = f"{self.project}:{self.rr}" if self.rr else None
         self.staging = not incident["inReview"]
+        self.embargoed = incident["crd"] is not None
```

Comment:
this is wrong :D It will have set CRD after embargo is lifted ...

So it should also compare dater time in CRD with current date time

Reply:
I replied to you in private . resolving this here if I not convinced you please reply me in private too